Gemfile

```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 gemspec
 
 group :test do
-  gem 'rom', git: 'https://github.com/rom-rb/rom.git', branch: 'master'
+  gem 'pry-byebug', platforms: :mri
```

Comment:
do we need it? I think it will be good to have almost the same gems set in all adapters